docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+as is allowed by these properties. This needs to be confirmed before this mapping is used. edm:dataProvider is used instead of
+edm:provider when the institution listed is not UTK as it  is defined as "The name or identifier of the organisation who
+contributes data indirectly to an aggregation service." Because UTK acts as a service hub for DPLA and it delivers data
+directly to this aggregator, it can be considered an edm:provider. This is defined as "The name or identifier of the
+organization who delivers data directly to an aggregation service (e.g. Europeana)."
+
+When UTK physically holds the material and created the record, the metadata resembles this `example record - acwiley:284 <https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A284/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource valueURI="http://id.loc.gov/authorities/names/n87808088">University of Tennessee, Knoxville. Libraries</recordContentSource>
+    </recordInfo>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider <http://id.loc.gov/authorities/names/n87808088> .
+
+Note that items held by UTK can also appear without a URI defining the institution as with `tdh:8803 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A8803/datastream/MODS/view>`_.
+It looks like we also have a misspelling here.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource>University of Tennesse Knoxville. Libraries</recordContentSource>
+    </recordInfo>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider "University of Tennesse Knoxville. Libraries" .
+
+Sometimes when the resource comes from another institution, that institution name is placed in <recordContentSource>. For instance,
+here's an `example record - cdf:70 <https://digital.lib.utk.edu/collections/islandora/object/cdf%3A70/datastream/MODS/view>`_.
+Here it is also coupled with an "Intermediate Provider" note. McClung's Egypt collection is also treated similarly.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource valueURI="http://id.loc.gov/authorities/names/no2017113530">Langston Hughes Library (Children's Defense Fund Haley Farm)</recordContentSource>
+    </recordInfo>
+    <note displayLabel="Intermediate Provider">University of Tennessee, Knoxville. Libraries</note>
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2017113530">Langston Hughes Library (Children's Defense Fund Haley Farm)</physicalLocation>
+    </location>
+
+For the purposes of DPLA, we only need the <recordContentSource> value and not also the <physicalLocation> value. Because
+these institutions are not directly contributing to DPLA, they are listed as an edm:dataProvider instead of an edm:provider.
+
```

Comment:
I wonder if we should separate this into two headings.  Something like:

### recordContentSource (University of Tennessee)

### recordContentSource (Not University of Tennessee)

Reply:
Great suggestion - I've just incorporated this into the mapping doc. Thank you!

Reply:
This was resolved in 2bd47b57b297b1805e1b9a7e6f517ce517eda056.